AcademyApp/Designables/UIButtonX.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UIButtonX.swift
+//  AcademyApp
+//
+//  Created by Peter Sevcik on 14/06/2019.
+//  Copyright © 2019 Peter Sevcik. All rights reserved.
+//
+
+import UIKit
+
+@IBDesignable
+class UIButtonX: UIButton {
```

Comment:
UIButtonX is not very good naming, try to find some better name, try to find common patterns in functionality and naming...
hints: are there similar buttons in app? Is it some kind of rounded button, etc